lib/internal/Magento/Framework/MessageQueue/Test/Unit/Topology/Config/QueueConfigItem/DataMapperTest.php

```diff
@@ -1,13 +1,16 @@
 <?php
+
+declare(strict_types=1);
+
```

Comment:
Please move it back to prev. position

Reply:
I have corrected rest of comments, but I don't understand this one

Reply:
Strict type declaration usually goes after copyright notice

Reply:
ok, corrected